lib/fetch-system-codes.js

```diff
@@ -16,8 +16,10 @@ function fetchSystemCodes(apiKey, page = 1, systemCodes = []) {
 		.then(response => {
 			// Grab the total number of pages of system codes
 			totalPages = getPagesFromCountHeader(response.headers.count);
-			// We're only interested in system code values
-			return response.body.map(entry => entry.value);
+			// We're only interested in system codes
+			return response.body
+				.filter(entry => entry.systemCode)
+				.map(entry => entry.systemCode);
 		})
```

Comment:
I assume the above is aiming to extract just the field named "systemCode" from the rather long list of fields returned by the v3/items/system call. However, due to a legacy issue with data uploads (too long to explain) the systemcode is actually held in the key field "dataItemID" not "systemCode".  You may find some "systemCode" fields but its no longer used.